pkg/utils/utils_test.go

```diff
@@ -808,11 +808,6 @@ var _ = Describe("Utils", Label("utils"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			Expect(f).To(Equal(filepath.Join(rootDir, file2)))
 		})
-		It("finds a matching file, first match wins file2", func() {
-			f, err := utils.FindFile(fs, rootDir, "/path/with/*aarch64/find*", "/path/with/*dle*/*me.*")
-			Expect(err).ShouldNot(HaveOccurred())
-			Expect(f).To(Equal(filepath.Join(rootDir, file2)))
-		})
```

Comment:
This is a duplicated test, the previous one is *exactly* the same code. I guess a copy & paste leftover